src/main/java/ch/njol/skript/config/Config.java

```diff
@@ -335,4 +337,35 @@ public int compareTo(@Nullable Config other) {
 		return fileName.compareTo(other.fileName);
 	}
 
+	@Override
+	public void invalidate() throws UnsupportedOperationException {
```

Comment:
```suggestion
	public void invalidate() {
```
It doesn't, right?

Reply:
It's custom to leave the `throws` on for overriding methods, but I can remove it if you want..